src/features/events/components/groupedEventsCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)